### ISSUE TITLE
Out of sync with the Apollo NPM packages again

### ISSUE THREAD

**Issue by csillag** [apollographql/meteor-integration#86]
When running with the latest Apollo packages, I get this warning:

```
WARNING: npm peer requirements (for apollo) not installed:
 - apollo-client@1.0.0-rc.2 installed, apollo-client@^0.10.0 needed
```

This package should be updated to deal with the situation...

**Comment by xavcz** [comment]
Thanks for the care @csillag 😌

There is an opened PR to be even more in sync with the Apollo NPM packages, solving this issue (see https://github.com/apollographql/meteor-integration/pull/85/files#diff-3f35f3c0b4e97fed39c8f5867a43acbdR7).

In addition of checking the NPM versions, it will allow you to use subscriptions in your app easily. You may be using them already? Your feedback on this PR is anyway welcome! 🎉

I will merge it this afternoon 👍 

*Note: 0.10.0 is the same as rc.0.*

**Comment by csillag** [comment]
> There is an opened PR to be even more in sync with the Apollo NPM packages,

:+1: 

> In addition of checking the NPM versions, it will allow you to use subscriptions in your app easily. You may be using them already? 

I tried, but I postponed it, after realizing that with the current state of events, it would be too much work. So not much to report on that front.

**Comment by csillag** [comment]
Indeed, this has been closed with 0.6.0.
